notebooks/05_Kubeflow_Pipeline/src/Pipeline_mnist/Dockerfile

```diff
@@ -0,0 +1,9 @@
+#This container contains your model and any helper scripts specific to your model.
```

Comment:
use lowercase here. change to `src/mnist/`. This is under pipeline folder already, Prefix `Pipeline_` is redundant

Reply:
good point, will fix it up.